apps/api/v2/src/ee/event-types-private-links/services/private-links.service.ts

```diff
@@ -0,0 +1,89 @@
+import { Injectable, NotFoundException, BadRequestException } from "@nestjs/common";
+
+import {
+  createPrivateLink,
+  deletePrivateLink,
+  getPrivateLinks,
+  updatePrivateLink,
+} from "@calcom/platform-libraries/private-links";
+import {
+  CreatePrivateLinkInput_2024_06_14,
+  PrivateLinkOutput_2024_06_14,
+  UpdatePrivateLinkInput_2024_06_14,
+} from "@calcom/platform-types";
+
+import { PrivateLinksInputService } from "./private-links-input.service";
+import { PrivateLinksOutputService } from "./private-links-output.service";
+
+@Injectable()
+export class PrivateLinksService {
```

Comment:
We still have `class PlatformPrivateLinksService` in addition to this in [packages/platform/libraries/private-links.ts](https://github.com/calcom/cal.com/pull/22943/files#diff-8f947e88352ebc01eed522adebb46e8e22068d22b4ff2df2a4ff3d93a5967edd)